### ISSUE TITLE
Some old default positions are missing in the new theme

### ISSUE THREAD

**Issue by carakas** [forkcms/forkcms#2044]
## Type

- Enhancement

## Problem description

Some core modules expect them to be there like the login widget or the search box

## Steps to reproduce

Install fork 5 and look in the backend at the pages

## Expected behavior / Proposed solutions

I think it would be best to add the missing position(s) back

**Comment by jeroendesloovere** [comment]
Probably fixed by https://github.com/forkcms/forkcms/pull/2039

**Comment by carakas** [comment]
@jeroendesloovere  nope :) Has nothing to do with that.

**Comment by jeroendesloovere** [comment]
Ok, indeed, templates are empty
```
  <templates>
    <template label="Default" path="Core/Layout/Templates/Default.html.twig" image="false">
      <positions>
        <position name="main" />
      </positions>
      <format>
        [main]
      </format>
    </template>
  </templates>
```

**Comment by jeroendesloovere** [comment]
Here is a screenshot what happens:
<img width="708" alt="schermafbeelding 2017-06-07 om 16 43 16" src="https://user-images.githubusercontent.com/588616/26884627-7de3c4c8-4ba0-11e7-9742-6b648909fea7.png">

**Comment by jeroendesloovere** [comment]
Also when visiting the website, every page redirects you to 404.

**Comment by jonasdekeukelaere** [comment]
The bug with the numbers in widget on the pages edit page has been fixed in #2063
